pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1072,7 +1072,7 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 				if value, ok := cniNetworks[iface.Name]; ok {
 					prefix := ""
 					// no error check, we assume that CNI type was set correctly
-					if net.Multus != nil {
+					if net.Npwg != nil || net.Multus != nil {
```

Comment:
Once those helper funcitons are added to schema.go, we could replace this (and similar if statements) with ```net.IsMultusOrNpwg()```

Reply:
done